CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 See full changelog for the OpenAPI schema (OAS) [here](https://github.com/plaid/plaid-openapi/blob/master/CHANGELOG.md).
 
+# 21.0.0
```

Comment:
Oh darn I forgot the generator update requires an update to the changelog. Do you need that from me or can you guys add it? I'm a little afraid to edit it at this stage and maybe screw up the release process.